core/common/src/main/java/alluxio/security/authorization/Permission.java

```diff
@@ -205,4 +205,27 @@ public int hashCode() {
   public String toString() {
     return mOwner + ":" + mGroup + ":" + mMode;
   }
+
+  /**
+   * Gets the owner bits from the mode in short format. E.g. mode 0775 returns 0007; mode 0500
+   * returns 0005.
+   *
+   * @param mode the mode in short format
+   * @return the owner bits in short format
+   */
+  public static short getOwnerBits(short mode) {
```

Comment:
actually, we do not need the `permission.get*Bits()` method, because we already have the `permission.getMode().get*Bits()`, we just need to add the corresponding `permission.getMode().set*Bits()` methods